.github/workflows/release.yaml

```diff
@@ -54,13 +58,33 @@ jobs:
           RELEASE: ${{ env.RELEASE_VERSION }}-ARM
         run: |
           cd opensearch-operator
-          pwd
-          ls
-          go version
-          go env
           aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/opsterio
           docker build -f Dockerfile-ARM -t public.ecr.aws/opsterio/opensearch-operator:$RELEASE ./
```

Comment:
Hey @idanl21 how about we use docker buildx, we should not be having 2 dockerfiles and 2 new tags for each architecture (arm and x64), we should have single tag that supports both architectures, Example https://github.com/opensearch-project/opensearch-ci/blob/main/.github/workflows/docker.yml#L26-L32, final image https://hub.docker.com/r/opensearchstaging/jenkins/tags (ECR also supports multi architecture) OS/ARCH should have amd64 and arm64.
Since we will be appVersion to our helm charts https://github.com/Opster/opensearch-k8s-operator/blob/main/charts/opensearch-operator/Chart.yaml#L24, having single tag will allow users not to change for each architecture.
thoughts @swoehrl-mw ?

Reply:
Hey @prudhvigodithi,m i tried to work with it when I created that work flow, but the end image didnt ran on ARM...
 you are welcome to upload a PR if you succeed to work it out :)